README.md

```diff
@@ -75,6 +77,67 @@ Our client library is modular. Each sub-library is a standalone implementation f
 | `realtime-{lang}`     | [`JS`](https://github.com/supabase/realtime-js)  | [`C#`](https://github.com/supabase/realtime-csharp) \| [`Dart`](https://github.com/supabase/realtime-dart) \| [`Python`](https://github.com/supabase/realtime-py) \| `Rust` \| `Ruby` \| `Go`                                                                                                                                        |
 | `gotrue-{lang}`       | [`JS`](https://github.com/supabase/gotrue-js)    | [`C#`](https://github.com/supabase/gotrue-csharp) \| [`Dart`](https://github.com/supabase/gotrue-dart) \| [`Python`](https://github.com/supabase/gotrue-py) \| `Rust` \| `Ruby` \| `Go`                                                                                                                                              |
 
+# Getting Started
```

Comment:
I'm worried that this might confuse people - they could think that it's "getting started with supabase" rather than "getting started with Contributing". Perhaps we can move all of this into a "CONTRIBUTING.md" file and link to it from the main readme?

Reply:
> yet another great PR thanks @Isaiah-Hamilton - have we sent you any swag yet? Could you please send your tshirt size and postal address to [swag@supabase.io](mailto:swag@supabase.io)?

Just sent the email and thank you for the swag

Reply:
Not really sure if this is the [CONTRIBUTING.md](https://github.com/supabase/.github/blob/main/CONTRIBUTING.md) that you are talking about or not.

Reply:
ah yeah you're right - let's actually make it DEVELOPERS.md, similar to this one:
https://github.com/angular/angular.js/blob/master/DEVELOPERS.md

We can put it directly inside this repo since (at the root)